test/producers/remediation_updates_test.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class RemediationUpdatesTest < ActiveSupport::TestCase
+  setup do
+    @issue_id = 'ssg:rhel7|short_profile_ref_id|rule_ref_id'
+  end
+
+  test 'handles missing kafka config' do
+    assert_nil RemediationUpdates.deliver(
+      host_id: '0001', issue_ids: [@issue_id]
+    )
+  end
```

Comment:
I noticed this as well. I think we need to implement at least a warning message when this happens (we can report a Jira for it) so it's clear what is happening.

Reply:
Feel free to create a new Jira/PR.

Reply:
Reported as https://issues.redhat.com/browse/RHICOMPL-947